src/entities/header.txt

```diff
@@ -2,15 +2,40 @@ THIS FILE IS AUTOGENERATED, DO NOT MODIFY
 
 Unvanquished entity definitions file for GTKRadiant and NetRadiant
 
-[TODO: Add license statement.]
+License:
+  Copyright (c) 2014, Daemon Developers
+  All rights reserved.
+  Redistribution and use in source and binary forms, with or without
+  modification, are permitted provided that the following conditions are met:
+   * Redistributions of source code must retain the above copyright
+     notice, this list of conditions and the following disclaimer.
+   * Redistributions in binary form must reproduce the above copyright
+     notice, this list of conditions and the following disclaimer in the
+     documentation and/or other materials provided with the distribution.
+   * Neither the name of the Daemon developers nor the
+     names of its contributors may be used to endorse or promote products
+     derived from this software without specific prior written permission.
+  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+  ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+  WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+  DISCLAIMED. IN NO EVENT SHALL DAEMON DEVELOPERS BE LIABLE FOR ANY
+  DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+  (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+  ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 
 Contributors:
-  1999 inolen, Suicide
+  2014 neumond, Viech
+
+Greetings:
+  We can thank these authors for the old Tremulous GPP's entities.def that was
+  used as documentation for that complete rewrite:
```

Comment:
Better: "We thank these authors for contributing to Tremulous' entity definition file that was used as documentation when creating this one."